dist/source/repo-creator/choose-repository.ts

```diff
@@ -121,13 +121,15 @@ export class ChooseRepository {
 	}
 	
 	protected search = () => {
-		this.clearSearchResults();
-		this.gitHub.search(this.searchInput).then(searchResults => {
+			this.gitHub.search(this.searchInput).then(searchResults => {
 			let resultTemplates = underscore(searchResults).map(searchResult => new Repository(searchResult.owner.login, searchResult.name, false, false, false, true));
 			this.mergeTemplates(resultTemplates);
-		}).catch((error: Error) => {
+			}).catch((error: Error) => {
 			this.eventAggregator.publish(error);
-		});
+			}).then(x => {
+			this.searching = false;
+			});
+			this.searching = true;
```

Comment:
This line and the line above should be un-indented once.  Same with a few other lines above.  It should look like this in the end:
```ts
	protected search = () => {
		this.gitHub.search(this.searchInput).then(searchResults => {
			let resultTemplates = underscore(searchResults).map(searchResult => new Repository(searchResult.owner.login, searchResult.name, false, false, false, true));
			this.mergeTemplates(resultTemplates);
		}).catch((error: Error) => {
			this.eventAggregator.publish(error);
		}).then(x => {
			this.searching = false;
		});
		this.searching = true;
	}
```

This will make the code easier to read for future readers as the indentations help define where the lambdas begin and end.